src/main/kotlin/sideChain/iroha/consumer/IrohaCryptoProviderImpl.kt

```diff
@@ -0,0 +1,38 @@
+package sideChain.iroha.consumer
+
+import Keypair
+import ModelCrypto
+import java.nio.file.Paths
+
+/**
+ * Provides with keypair loaded from files
+ * @param pubkeyPath path to file with public key
+ * @param privkeyPath path to file with private key
+ */
+class IrohaCryptoProviderImpl(pubkeyPath: String, privkeyPath: String) : IrohaCryptoProvider {
+
+    /** Iroha keypair */
+    override val keypair: Keypair
+
+    init {
+        // Load keypair from existing files
+        val crypto = ModelCrypto()
+        keypair = crypto.convertFromExisting(
+            readKeyFromFile(pubkeyPath),
+            readKeyFromFile(privkeyPath)
+        )
+    }
+
+    /**
+     * Read key from file specified by [path]
+     */
+    private fun readKeyFromFile(path: String): String {
```

Comment:
This method doesn't relate to keys in terms of types. We have to move it into util package also. And use it here as a dependency.